java/core/src/java/org/apache/orc/impl/ColumnStatisticsImpl.java

```diff
@@ -170,7 +165,198 @@ public int hashCode() {
     }
   }
 
-  private static final class IntegerStatisticsImpl extends ColumnStatisticsImpl
+  /**
+   * Column statistics for List and Map types.
+   */
+  private static final class CollectionColumnStatisticsImpl extends ColumnStatisticsImpl
+      implements CollectionColumnStatistics {
+
+    protected long minimum = Long.MAX_VALUE;
+    protected long maximum = Long.MIN_VALUE;
```

Comment:
I'd suggest using 0 here, so that we don't accidentally use a negative length.

Reply:
Right, Fixed it.